src/shared/exec-util.h

```diff
@@ -49,8 +49,9 @@ typedef enum ExecCommandFlags {
         EXEC_COMMAND_FULLY_PRIVILEGED = 1 << 1,
         EXEC_COMMAND_NO_SETUID        = 1 << 2,
         EXEC_COMMAND_NO_ENV_EXPAND    = 1 << 3,
+        EXEC_COMMAND_PREFIX_SHELL     = 1 << 4,
```

Comment:
hmm, "prefix shell" is a bit misleading, no? sounds as if we'd just prefix /bin/sh to the argv list. but we don't. we replace the argv list via "sh -c" and then a concatenated version of the original argv, afaics.

hence "prefix" is weird. 

maybe call the flag EXEC_COMMAND_VIA_SHELL